docs/messaging/messaging-spans.md

```diff
@@ -153,36 +153,35 @@ in such a way that it cannot be changed by intermediaries.
 
 ### Span name
 
-The span name SHOULD be set to the message destination name and the name of the operation being performed (as captured in [`messaging.operation.name`](../attributes-registry/messaging.md)) in the following format:
+Messaging spans SHOULD follow the overall [guidelines for span names](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/trace/api.md#span).
 
-```
-<destination name> <operation name>
-```
+<!-- markdown-link-check-disable -->
+<!-- HTML anchors are not supported https://github.com/tcort/markdown-link-check/issues/225-->
+The **span name** SHOULD be `{messaging.operation.name} {destination}` (see below for the exact definition of the [`{destination}`](#destination-placeholder) placeholder).
```

Comment:
What's the main reason for turning this around (the operation name before the destination)? Consistency with DB?

Reply:
Yes, with DB and HTTP - they all share the same pattern {verb} {target} which seems to be more English-friendly.